docs/user-guide/cdf_attribute_management.rst

```diff
@@ -7,26 +7,26 @@ Using SAMMI CDF Attribute Manager for Metadata Attributes
 Overview
 ========
 
-The :py:class:`~sammi.cdf_attribute_manager.CdfAttributeManager` class provides an interface to configure how metadata attributes are formatted in SWxSOC affiliated data products. 
-The class represents a schema for metadata attribute requirements, validation, and formatting. 
+The :py:class:`~sammi.cdf_attribute_manager.CdfAttributeManager` class provides an interface to configure how metadata attributes are formatted in SWxSOC affiliated data products.
```

Comment:
Should this be made more general rather than saying it's for the SWxSOC? I think it's a little confusing to see this and have examples from IMAP further down.